src/assets/sass/protocol/includes/_themes.scss

```diff
@@ -6,6 +6,34 @@
 // themes. Mixins and functions can draw from these variable maps, swapping to
 // a different map based on the value of the global $brand-theme variable.
 
+//* -------------------------------------------------------------------------- */
+// Vertical and Horizontal Spacing
+
+
+$theme-spacing-firefox: (
```

Comment:
Let's rename this `$theme-spacing` so it's not tied explicitly to the brand. 

In the near future we'll define a condensed spacing and probably give set it with a separate theme variable like we do with the condensed type scale, but for now just defining these units gives us a good start.